README.md

```diff
@@ -114,7 +119,9 @@ helm upgrade --install -n stackrox --create-namespace stackrox-central-services
   --set central.persistence.none="true"
 ```
 
-If you're installing on a single node cluster, or the default installation results in pods stuck pending due to lack of resources, use the following command instead to reduce stackrox-central-services resource requirements. Keep in mind that these reduced resource settings are not suited for a production setup.
+#### Central Installation in Clusters With Limited Resources
+
+If you're installing in a single node cluster, or the default installation results in pods stuck pending due to lack of resources, use the following command instead to reduce stackrox-central-services resource requirements. Keep in mind that these reduced resource settings are not suited for a production setup.
```

Comment:
The change here is the 4th word: "on" vs "in".  I realized the headers I had removed used "installation _in_", so I changed this new text to match that.